ui/lib/pki/addon/components/page/pki-issuer-details.hbs

```diff
@@ -35,7 +35,13 @@
         Sign Intermediate
       </ToolbarLink>
     {{/if}}
-    <BasicDropdown @class="popup-menu" @horizontalPosition="auto-right" @verticalPosition="below" as |D|>
+    <BasicDropdown
```

Comment:
<img width="271" alt="Screenshot 2024-05-03 at 10 41 20" src="https://github.com/hashicorp/vault/assets/82459713/d79768d6-9a79-4d4c-9936-a086b4b839ca">

I don't know why it's centered like that, but these should be replaced by HDS::Dropdown soon anyway. This is what it looks like in the binary too